app/views/home/failure.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="py1 mb3 center">
+  <h2>Authentication from Login.gov was not successful! :(</h2>
```

Comment:
Sorry, it's a WIP. Have update soon!

On Thu, Jun 30, 2016 at 4:02 PM, Sam Clegg <notifications@github.com> wrote:

> In app/views/home/failure.html.erb
> <https://github.com/18F/identity-sp-rails/pull/14#discussion_r69224917>:
>
> > @@ -0,0 +1,18 @@
> > +<div class="py1 mb3 center">
> > +  <h2>Authentication from Login.gov was not successful! :(</h2>
>
> Can we get failure messages from omniauth? It think its good to display
> them here.. but we can just put them in the logs if you prefer.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/identity-sp-rails/pull/14/files/c4a59f4a0ceb718b6bd1973096dfb060e80fe5df#r69224917>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABQi_4Ok2qM5lmJFZzEVwoabgjrillFCks5qREsEgaJpZM4JBbiT>
> .
>